lib/team.rb

```diff
@@ -1,16 +1,16 @@
 class Team
 
   attr_reader :team_id,
-              :franchise_id,
-              :team_name,
+              :franchiseid,
```

Comment:
Good catch here, we'll have to run through the rest of our attrs to make sure this isn't happening anywhere else